apps/mull-ui/src/app/components/event-page-header/event-page-header.tsx

```diff
@@ -31,14 +38,9 @@ export const EventPageHeader = ({ event, prevPage }: EventPageHeaderProps) => {
 
   return (
     <div className="event-page-header">
-      <MullBackButton>{prevPage}</MullBackButton>
+      <MullBackButton onClick={onBackButtonClick}>{prevPage}</MullBackButton>
```

Comment:
This doesn't read so well `onClick={onBackButtonClick}`. I usually see something like `onClick={handleBackButton}`. Curious what you think

Reply:
If that is what's most common, then I'll rename it accordingly 😄